frontend/src/lib/k8s/api/v2/useKubeObjectList.ts

```diff
@@ -114,44 +114,80 @@ function useWatchKubeObjectLists<K extends KubeObject>({
 }) {
   const client = useQueryClient();
 
-  const connections = useMemo(() => {
-    if (!endpoint) return [];
+  // We'll watch only the first list since backend handles multiplexing
+  const firstList = lists[0];
```

Comment:
this is not correct, we need to start watching all of lists. for example when we have two allowedNamespaces enabled ('a' and 'b') we'll receive two objects `[{ cluster: 'cluster', namespace: 'a', resourceVersion: 0 }, { cluster: 'cluster', namespace: 'b', resourceVersion: 0 }]` and both connections need to be made